src/screens/Swap2/FormSelection/SelectFeesScreen.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+import { BigNumber } from "bignumber.js";
+
+import React, { useCallback } from "react";
+import { StyleSheet, SafeAreaView } from "react-native";
+
+import type {
+  Account,
+  AccountLikeArray,
+} from "@ledgerhq/live-common/lib/types";
+import { TrackScreen } from "../../../analytics";
+import { ScreenName } from "../../../const";
+
+import type { SwapRouteParams } from "..";
+import SendRowsFee from "../../../components/SendRowsFee";
+import NavigationScrollView from "../../../components/NavigationScrollView";
+
+type Props = {
+  accounts: Account[],
+  allAccounts: AccountLikeArray,
+  navigation: any,
+  route: { params: SwapRouteParams },
+};
+
+export default function SelectFees({ navigation, route }: Props) {
+  const { transaction } = route.params;
+  const { account, parentAccount } = route.params;
+
+  // Tmp: fixed networkInfo while waiting for ll-common to build complete transaction info
```

Comment:
```suggestion
  // FIXME : Tmp fixed networkInfo while waiting for ll-common to build complete transaction info
```

Reply:
Removing the whole Fixme code, it was a mistake on my side